scripts/ci/run_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e -o pipefail
+set -x
+
+PROJECT=${project:-spryker}
```

Comment:
same here, so this is the `project` from travis config? Please make it also upper case